docs/Homebrew-brew-Maintainer-Guide.md

```diff
@@ -10,110 +9,64 @@ Using `gh pr checkout NUMBER` is a super easy way to check out a PR branch using
-- `codecov/patch` and `codecov/project`: These show the Codecov report for the PR.
-  See the ["`brew tests` and Codecov" section below](#brew-tests-and-codecov) for more info about Codecov.
-- `CI / vendored gems (Linux)`: This checks whether there was a change to the vendored gems on Linux that needs to be
-  committed to the PR branch.
+- `Triage / review`: This verifies that the PR has been open for long enough. See the [Automatic approvals](#automatic-approvals) section above for more information about automatic approvals.
+- `Vendor Gems / vendor-gems`: This is skipped except for dependabot PRs. It updates the RBI files to match any new/changed dependencies. See [Type Checking With Sorbet](Typechecking.md) for more information about RBI files and typechecking.
+- `Codecov / codecov/patch` and `codecov/project`: These show the Codecov report for the PR. See the [`brew tests` and Codecov](#brew-tests-and-codecov) section below for more info about Codecov.
+- `CI / vendored gems`: This checks whether there was a change to the vendored gems on Linux that needs to be committed to the PR branch.
 - `CI / test default formula (Linux)`: This runs `brew test-bot` on Linux to ensure it still works as expected.
-- `CI / syntax`: This is run first to check whether the PR passes `brew style` and `brew typecheck`. If this job fails the
-  following jobs will not run.
-- `CI / tap syntax (Linux)`: This runs `brew style` and `brew audit` on all official taps
-  (note that although this has Linux in its name, it does check `Homebrew/homebrew-core` and all cask repos).
-- `CI / docker`: This builds and deploys a new Homebrew Docker image.
-- `CI / test everything (macOS)`: This runs several checks on macOS including `brew tests`, `brew update-tests`,
-  `brew test-bot --only-formulae --test-default-formula`, `brew readall` and `brew doctor`.
-- `CI / tests (no-compatibility mode)`, `CI / tests (generic OS)` and `CI / tests (Linux)`: These run
-  `brew tests` with various options on Linux.
+- `CI / syntax`: This is run first to check whether the PR passes `brew style` and `brew typecheck`. If this job fails the following jobs will not run.
+- `CI / tap syntax`: This runs `brew style` and `brew audit` on all official taps (note that although this runs on Linux, it does check all cask repos).
+- `CI / docker`: This builds and deploys a new Homebrew Docker image to GitHub Packages and Docker Hub.
+- `CI / test everything (macOS)`: This runs several checks on macOS including `brew tests`, `brew update-tests`, `brew test-bot --only-formulae --test-default-formula`, `brew readall` and `brew doctor`.
+- `CI / tests (no-compatibility mode)`, `CI / tests (generic OS)` and `CI / tests (Linux)`: These run `brew tests` with various options on Linux.
+- `Documentation CI / linting` and `rubydoc`: These check the prose and formatting of the written documentation, and verify the [rubydoc API documentation](https://rubydoc.brew.sh) is built without issue.
```

Comment:
```suggestion
- `Documentation CI / linting` and `rubydoc`: These check the prose and formatting of the written documentation, and verify the [rubydoc API documentation](https://rubydoc.brew.sh) can be built without issue.
```

Maybe? The actual website build is performed elsewhere and at a later time.

Reply:
True, although it won't actually return a failure until [`--fail-on-warning`](https://github.com/Homebrew/brew/blob/d9ef4545282f0f98664594fb4991b9602a9f27ea/.github/workflows/docs.yml#L67) is enabled (and the failures it produces are fixed) — stay tuned.